Library/ViewModels/DiscoveryViewModel.swift

```diff
@@ -126,6 +126,13 @@ public final class DiscoveryViewModel: DiscoveryViewModelType, DiscoveryViewMode
       .takeWhen(self.pageTransitionCompletedProperty.signal.filter(isTrue))
       .map { sorts[$0] }
 
+    let prevSortedPage = Signal.merge(
+      swipeToSort,
+      self.sortPagerSelectedSortProperty.signal.skipNil()
+    )
+    .combinePrevious(.magic)
+    .map { $0.0 }
```

Comment:
I think `$0.0` is a bit hard to read. I would opt for something that is more verbose here so it's easier to understand for someone who did not write the code.

Reply:
Yeah. I understand the concern, I opted for a more function programming approach, a more verbose option would be to open up the map function or use `map(first)`, I have opted for `map(first)`